.travis.yml

```
@@ -16,6 +16,8 @@ matrix:
   - python: 3.8
     env: PYV=3.8
   - python: pypy3
```

```suggestion
  - python: pypy3
    dist: bionic
```

IIRC this would work?